src/backend/schemas/agent.py

```diff
@@ -7,6 +8,12 @@
 from backend.schemas.deployment import ModelSimple as ModelSchema
 
 
+class AgentVisibility(str, Enum):
+    PRIVATE = "private"
+    PUBLIC = "public"
+    ALL = "all"
```

Comment:
isn't the same, public / all?

Reply:
`All` is supposed to be `public + private`. So if a user has private agents, you could get all the agents they have access to by using `All`.